### PR TITLE
Add bottom margin to spans in hovers

### LINKED ISSUE #101496: Add bottom margin to issue hover
Refs: microsoft/vscode-pull-request-github#1937

Hover over na issue
Notice how there is not a lot of bottom padding -> this gives a wrong visual impression to the user that the bottom is clipped and that he should scroll down - which is not true.
I suggest to add 2px more padding to the bottom.

Screenshot 2020-07-01 at 11 35 22

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #101496

### CODE REVIEW

**Comment by alexr00** [comment]
Some examples of what this looks like (original in the linked issue):
image
image

File: src/vs/base/browser/ui/hover/hover.css [hunk 1]
```diff
@@ -131,3 +131,8 @@
 	border-bottom: 1px solid transparent;
 	text-underline-position: under;
 }
+
+.monaco-hover .markdown-hover .hover-contents span {
+	margin-bottom: 4px;
```
Comment (Tyriar) [inline src/vs/base/browser/ui/hover/hover.css:136]: Can you add a comment about why this is here?

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/base/browser/ui/hover/hover.css (hunk 1 above)
Comment (alexr00) [reply src/vs/base/browser/ui/hover/hover.css:136]: Added!

**Review by Tyriar — APPROVED** [review]

**Review by mjbvz — APPROVED** [review]

File: src/vs/base/browser/ui/hover/hover.css [hunk 2]
```diff
@@ -131,3 +131,9 @@
 	border-bottom: 1px solid transparent;
 	text-underline-position: under;
 }
+
+/** Spans in markdown hovers need a margin-bottom to avoid looking cramped: https://github.com/microsoft/vscode/issues/101496 **/
+.monaco-hover .markdown-hover .hover-contents span {
+	margin-bottom: 4px;
```
Comment (mjbvz) [inline src/vs/base/browser/ui/hover/hover.css:137]: Maybe this should use `em` so it is updated automatically if the font size changes

**Review by andrewchoi5 — APPROVED** [review]

File: src/vs/base/browser/ui/hover/hover.css (hunk 1 above)
Comment (alexr00) [reply src/vs/base/browser/ui/hover/hover.css:137]: Given that almost all of the other sizes are `px`, I think it is safe to keep it `px`.

**Pull request merged**